src/frontend/js/data/useStaticFilters/index.tsx

```diff
@@ -15,6 +16,14 @@ const coursesConfig: FilterDefinition = {
   position: 99,
 };
 
+const messages = defineMessages({
+  courses: {
+    defaultMessage: 'Courses',
+    description: 'localized human_name label for coursesConfig filter definition',
```

Comment:
Maybe more a name than a definition?
```suggestion
    description: 'localized human_name label for coursesConfig filter name',
```